package.json

```diff
@@ -22,9 +22,7 @@
     "react": "^16.4.1",
     "react-content-loader": "^3.1.2",
     "react-dom": "^16.4.1",
-    "react-router-dom": "^4.3.1",
-    "react-winjs": "git+https://github.com/TECLIB/react-winjs.git#develop",
```

Comment:
wow!

Reply:
Yes, I have replaced the components of winjs with fabric